src/OrbitQt/HistogramWidget.cpp

```diff
@@ -87,74 +93,129 @@ static void DrawVerticalLine(QPainter& painter, const QPoint& start, int length)
   painter.drawLine(start, {start.x(), start.y() + length});
 }
 
+struct Ticks {
+  std::vector<QString> labels;
+  std::vector<double> values;
```

Comment:
I see. I would have them together generating directly the pairs (there are not too many labels so this can be a problem in any way), but I don't have a strong opinion about that, it's just I don't like to have two vectors that had a hidden relationship. I would rather use some kind of map (labels, values)